### ISSUE TITLE
Needs significant updating

### ISSUE THREAD

**Issue by RangerDjanger** [microsoft/MCW-Enterprise-class-networking#24]
The HOL needs much updating.

Running the instructions exactly as they are does not lead to a successull lab. The biggest issues in instructions are:
- Pre-prepared VM/Azure environment pre-creates RGs in different regions. Each some students get Australia East, some get Australia South East. Instructions refer to "South Central".  
- This effects the ARM template provisioning. We had to update the ARM template to not use resourcegroup.getlocation()
- The Lab doesnt actualy have instructions to validate the lab. I.e. jump onto the VM on-prem and browse to the LB.
- The VNET peering instructions are not complete. 
- The lab is too long - can be optimised. Its challenging to be completed within 2.5-3 hours 

1.  Instructions updated to not create resource groups if using a pre-provisioned Lab environment.
2.  Update Lab instructions to confirm which region the pre-provisioned Lab VM has deployed resource groups - or change the Cloud Shop ARM template to remove resourcegroup.getocation.
3. Update lab secnario text and remove Barracuda firewall. Replace with Azure firewall.
4.  Change 'perimeter' subnet screen shots with AzureFirewall.
5. Update architecture diagram with Azure Firewall (not VM).
6. Update architecture diagram to indicate correct routes between Firewall and subnets.
7. Route Table - BGP screen shots out of date
8. Az Firewall UI's/instructions need slight update.

**Comment by DawnmarieDesJardins** [comment]
@mwasham @Jeff-Opsgility 
Hi Michael,
This workshop was updated 11/18 & is scheduled for a test/fix 5/19. In our meeting Monday, let's revisit the current review cycle for this one.

**Comment by polichtm** [comment]
•	Pre-prepared VM/Azure environment pre-creates RGs in different regions. Each some students get Australia East, some get Australia South East. Instructions refer to "South Central".
•	This effects the ARM template provisioning. We had to update the ARM template to not use resourcegroup.getlocation()
o	The above two should be addressed with whoever is providing the pre-provisioned environments.
•	The Lab doesnt actualy have instructions to validate the lab. I.e. jump onto the VM on-prem and browse to the LB.
o	This has been added
•	The VNET peering instructions are not complete.
o	updated
•	The lab is too long - can be optimised. Its challenging to be completed within 2.5-3 hours
o	This would require changing the scenario 
•	Instructions updated to not create resource groups if using a pre-provisioned Lab environment.
o	This lab is written assuming the user performs all of the steps in the lab (which includes creating the resource groups). e they are pre-deploying. The only piece that can be pre-deployed without altering the flow of the lab is the LABVM. 
•	Update Lab instructions to confirm which region the pre-provisioned Lab VM has deployed resource groups - or change the Cloud Shop ARM template to remove resourcegroup.getocation.
o	Students/trainers need to verify the pre-deployed resource group locations if they are using a pre-provisioned environment?
•	Update lab secnario text and remove Barracuda firewall. Replace with Azure firewall.
o	updated
•	Change 'perimeter' subnet screen shots with AzureFirewall.
o	updated
•	Update architecture diagram with Azure Firewall (not VM).
o	updated
•	Update architecture diagram to indicate correct routes between Firewall and subnets.
o	updated
•	Route Table - BGP screen shots out of date
o	updated
•	Az Firewall UI's/instructions need slight update.
o	updated
